Detectors/FIT/reconstruction/src/RecPoints.cxx

```diff
@@ -29,24 +30,22 @@ void RecPoints::FillFromDigits(const Digit& digit)
   Float_t cfd[nMCPs] = {}, amp[nMCPs] = {};
   Float_t sideAtime = 0, sideCtime = 0;
 
-  Float_t eventTimeFromDigits = digit.getTime();
-  constexpr Double_t BC_clk = 25.;                //ns event clk lenght
-  constexpr Double_t BC_clk_center = BC_clk / 2.; // clk center
-  Int_t nClk = floor(eventTimeFromDigits / BC_clk);
-  Double_t BCEventTime = eventTimeFromDigits - BC_clk * nClk;
-  mEventTime = BC_clk_center + BCEventTime;
-  //   std::cout << " Event Time from digits" << eventTimeFromDigits <<" BCEventTime "<<BCEventTime<< " mEveentTime "<< mEventTime<<std::endl;
+  mEventTime = digit.getTime();
+  mBC = digit.getBC();
+  mOrbit = digit.getOrbit();
+
+  Float_t BCEventTime = 12.5;
+
   for (const auto& d : digit.getChDgData()) {
     Int_t mcp = d.ChId;
-    cfd[mcp] = d.CFDTime /*- BC_clk_center - BCEventTime*/;
+    cfd[mcp] = d.CFDTime - mEventTime - BCEventTime;
```

Comment:
Does the mEventTime here correspond to the MC-truth time supplied by the interaction sampler during the digitization? The reconstruction should not use MC-truth times, should not you derive it from the orbit and BC?

Reply:
Dear Ruben and Sandro,
I fixed code according your suggestion.
May I ask to merge branch now? Next step will be to separate T0 and V0 within FIT directory. 
We need to start from recent dev branch with all these developments inside.

Best regards
Alla